acceptance-tests/src/test/java/org/eclipse/collections/impl/forkjoin/FJIterateAcceptanceTest.java

```diff
@@ -60,6 +60,8 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.junit.Assert.assertThrows;
```

Comment:
This is a JUnit 4 import. It looks like the sort of thing we did in other modules but I'm wondering if it's intentional that it's part of the same PR. It means this is not the last one, but includes prep for one more.